app/print.py

```diff
@@ -0,0 +1,21 @@
+from abc import ABC, abstractmethod
+
+from .display import DisplayConsole, DisplayReverse
+
+
+class Print(ABC):
+    @abstractmethod
+    def print_book(self, title: str, content: str) -> None:
+        pass
+
+
+class PrintConsole(Print, DisplayConsole):
+    def print_book(self, title: str, content: str) -> None:
+        print(f"Printing the book: {title}...")
+        self.display(content)
+
+
+class PrintReverse(Print, DisplayReverse):
```

Comment:
SRP. These responsibilities should be separated. You make tight responsibilites between displaying and printing, that should be either displaying or printing.